### ISSUE TITLE
Build fails on Windows 10 with Qt 5.6

### ISSUE THREAD

**Issue by mitchcurtis** [sqlitebrowser/sqlitebrowser#525]
I checked out the source, ran `qmake` in the source directory, then `nmake`, and got this output:

```
moc_ScintillaQt.cpp
Generating Code...
        lib /NOLOGO  /OUT:release\qscintilla2.lib @C:\Users\Mitch\AppData\Local\Temp\nmBA84.tmp
        cd src\ && ( if not exist Makefile C:\dev\qt5.6-debug\qtbase\bin\qmake C:\dev\sqlitebrowser\src\src.pro -o Makefile ) && "C:\dev\Microsoft Visual Studio 14.0\VC\BIN\amd64\nmake.exe" -f Makefile
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_ru.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_de.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_fr.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_zh.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_zh_TW.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_pt_BR.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_en_GB.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_ko_KR.qm'
RCC: Warning: No resources in 'translations\translations.qrc'.
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_ru.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_de.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_fr.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_zh.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_zh_TW.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_pt_BR.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_en_GB.qm'
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_ko_KR.qm'
RCC: Warning: No resources in 'translations\translations.qrc'.

Microsoft (R) Program Maintenance Utility Version 14.00.23026.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        "C:\dev\Microsoft Visual Studio 14.0\VC\BIN\amd64\nmake.exe" -f Makefile.Debug

Microsoft (R) Program Maintenance Utility Version 14.00.23026.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        C:/dev/qt5.6-debug/qtbase/bin/lrelease.exe translations\sqlb_zh.ts -qm translations/sqlb_zh.qm
        C:/dev/qt5.6-debug/qtbase/bin/lrelease.exe translations\sqlb_zh_TW.ts -qm translations/sqlb_zh_TW.qm
        C:/dev/qt5.6-debug/qtbase/bin/lrelease.exe translations\sqlb_de.ts -qm translations/sqlb_de.qm
        C:/dev/qt5.6-debug/qtbase/bin/lrelease.exe translations\sqlb_fr.ts -qm translations/sqlb_fr.qm
        C:/dev/qt5.6-debug/qtbase/bin/lrelease.exe translations\sqlb_ru.ts -qm translations/sqlb_ru.qm
        C:/dev/qt5.6-debug/qtbase/bin/lrelease.exe translations\sqlb_pt_BR.ts -qm translations/sqlb_pt_BR.qm
        C:/dev/qt5.6-debug/qtbase/bin/lrelease.exe translations\sqlb_en_GB.ts -qm translations/sqlb_en_GB.qm
        C:/dev/qt5.6-debug/qtbase/bin/lrelease.exe translations\sqlb_ko_KR.ts -qm translations/sqlb_ko_KR.qm
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe MainWindow.ui -o .ui\ui_MainWindow.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe CreateIndexDialog.ui -o .ui\ui_CreateIndexDialog.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe AboutDialog.ui -o .ui\ui_AboutDialog.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe EditTableDialog.ui -o .ui\ui_EditTableDialog.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe PreferencesDialog.ui -o .ui\ui_PreferencesDialog.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe EditDialog.ui -o .ui\ui_EditDialog.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe ExportCsvDialog.ui -o .ui\ui_ExportCsvDialog.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe ImportCsvDialog.ui -o .ui\ui_ImportCsvDialog.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe SqlExecutionArea.ui -o .ui\ui_SqlExecutionArea.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe VacuumDialog.ui -o .ui\ui_VacuumDialog.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe CipherDialog.ui -o .ui\ui_CipherDialog.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe ExportSqlDialog.ui -o .ui\ui_ExportSqlDialog.h
        (set QT_PLUGIN_PATH=C:\dev\qt5.6-debug\qtbase\plugins) & (set PATH=C:\dev\qt5.6-debug\qtbase\bin;%PATH:)=^)%) & C:\dev\qt5.6-debug\qtbase\bin\uic.exe ColumnDisplayFormatDialog.ui -o .ui\ui_ColumnDisplayFormatDialog.h
        C:\dev\qt5.6-debug\qtbase\bin\rcc.exe -name icons icons\icons.qrc -o debug\qrc_icons.cpp
        C:\dev\qt5.6-debug\qtbase\bin\rcc.exe -name flags translations\flags\flags.qrc -o debug\qrc_flags.cpp
        C:\dev\qt5.6-debug\qtbase\bin\rcc.exe -name translations translations\translations.qrc -o debug\qrc_translations.cpp
RCC: Error in 'translations\translations.qrc': Cannot find file 'sqlb_ru.qm'
NMAKE : fatal error U1077: 'C:\dev\qt5.6-debug\qtbase\bin\rcc.exe' : return code '0x1'
Stop.
NMAKE : fatal error U1077: '"C:\dev\Microsoft Visual Studio 14.0\VC\BIN\amd64\nmake.exe"' : return code '0x2'
Stop.
NMAKE : fatal error U1077: 'cd' : return code '0x2'
Stop.
```

**Comment by glawrence** [comment]
Have you managed to follow the same process on Windows 7?

**Comment by mitchcurtis** [comment]
I don't have Windows 7.

**Comment by glawrence** [comment]
I have a Windows 8.1 VM that I have been trying to build DB4S on for a while now, with no success as yet. However I notice you have VS2015 and I don't think Qt is compatible with VS2015 yet, so you might need to try VS2013 instead.

**Comment by justinclift** [comment]
As another thought, that might be a race condition in the build showing up.  If you run nmake again after it fails like that, does the compile then finish?

**Comment by mitchcurtis** [comment]
@justinclift Nope.

**Comment by justinclift** [comment]
So much for hoping for an easy solution. :grin:

**Comment by glawrence** [comment]
It has just dawned on me, Qt 5.6 has not been released yet and I believe it will work with VS2015, however if you are at the bleeding edge then you have less chance of success than using supported releases. Just saying, you're welcome to keep trying though

**Comment by mitchcurtis** [comment]
@glawrence It's cool; I ended up just downloading the binaries.

**Comment by glawrence** [comment]
@mitchcurtis good plan! If you are happy, can you close this issue, or let me know and I'll do it, thanks

**Comment by mitchcurtis** [comment]
But it's not fixed?

**Comment by glawrence** [comment]
True but the nightly build process works as does the release build. Our Windows build process is not well defined but if it was I doubt it would cover Qt 5.6

**Comment by mitchcurtis** [comment]
So it's not planned to support Qt 5.6?

**Comment by glawrence** [comment]
I would expect so as I believe 5.6 will be the LTS release, however as 5.6 is only at Release Candidate right not, then it will be a while. Once we have a documented Windows build process then changing Qt version will be easier, right now you would need to do this on Linux and then copy the Windows output to Windows 10.

**Comment by justinclift** [comment]
Yeah, it's kind of a hard call.  While it would be nice to support Qt 5.6 right now... we just don't have any developer around that are interested in this just yet.  When we go to make a new release after Qt 5.6 is out, it's reasonably likely we'll hit the same issues you're reporting, and get them fixed then.

So, not the speediest of approaches. :wink:

That being said, if you have the time/interest to work out what's wrong, and submit a Pull Request with the fixes, we'd gratefully accept it. :grin:

**Comment by MKleusberg** [comment]
Closing this because there were no updates for four years and it doesn't make much sense to investigate a build file for such an old version. That said, if support for Qt 5.6 is still needed we will definitely try to fix our code to make it work with Qt 5.6. For that, can you try to compile again with our latest code from the master branch and open a new issue if there are any errors? We can then adjust our current code base which should be much easier :smile:
